app/assess/templates/landing.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{%- from 'govuk_frontend_jinja/components/inset-text/macro.html' import govukInsetText -%}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{% import 'components/projects_table/projects_table.html' as projects_table-%}
+
+{% block content %}
+  <div>
+      <p class="govuk-heading-s">Morning Jane Doe</p>
```

Comment:
Good to see all gov uk styling, does the parent div need anything? Just wondering.

Reply:
Good question - will check

Reply:
Removed